applications/FluidDynamicsApplication/custom_elements/symbolic_stokes.cpp

```diff
@@ -0,0 +1,4943 @@
+            lhs(30,14)=DN(7,0)*clhs88 + DN(7,1)*clhs227 + DN(7,2)*clhs307 + clhs698;
+            lhs(30,15)=-clhs714;
+            lhs(30,16)=DN(7,0)*clhs94 + DN(7,1)*clhs231 + DN(7,2)*clhs312 + clhs739;
+            lhs(30,17)=DN(7,0)*clhs104 + DN(7,1)*clhs234 + DN(7,2)*clhs314 + clhs758;
+            lhs(30,18)=DN(7,0)*clhs110 + DN(7,1)*clhs240 + DN(7,2)*clhs315 + clhs776;
+            lhs(30,19)=-clhs788;
+            lhs(30,20)=DN(7,0)*clhs116 + DN(7,1)*clhs244 + DN(7,2)*clhs320 + clhs807;
+            lhs(30,21)=DN(7,0)*clhs126 + DN(7,1)*clhs247 + DN(7,2)*clhs322 + clhs821;
+            lhs(30,22)=DN(7,0)*clhs132 + DN(7,1)*clhs253 + DN(7,2)*clhs323 + clhs834;
+            lhs(30,23)=-clhs842;
+            lhs(30,24)=DN(7,0)*clhs138 + DN(7,1)*clhs257 + DN(7,2)*clhs328 + clhs854;
+            lhs(30,25)=DN(7,0)*clhs148 + DN(7,1)*clhs260 + DN(7,2)*clhs330 + clhs863;
+            lhs(30,26)=DN(7,0)*clhs154 + DN(7,1)*clhs266 + DN(7,2)*clhs331 + clhs871;
+            lhs(30,27)=-clhs875;
+            lhs(30,28)=DN(7,0)*clhs160 + DN(7,1)*clhs270 + DN(7,2)*clhs336 + clhs881;
+            lhs(30,29)=DN(7,0)*clhs170 + DN(7,1)*clhs273 + DN(7,2)*clhs338 + clhs884;
+            lhs(30,30)=DN(7,0)*clhs176 + DN(7,1)*clhs279 + DN(7,2)*clhs339 + clhs878 + clhs886*mu;
+            lhs(30,31)=-clhs887;
+            lhs(31,0)=clhs177 + clhs344*clhs370;
+            lhs(31,1)=clhs280 + clhs344*clhs371;
+            lhs(31,2)=clhs342 + clhs344*clhs372;
+            lhs(31,3)=clhs373;
+            lhs(31,4)=clhs344*clhs504 + clhs416;
+            lhs(31,5)=clhs344*clhs505 + clhs450;
+            lhs(31,6)=clhs344*clhs506 + clhs483;
+            lhs(31,7)=clhs507;
+            lhs(31,8)=clhs344*clhs618 + clhs544;
+            lhs(31,9)=clhs344*clhs619 + clhs573;
+            lhs(31,10)=clhs344*clhs620 + clhs601;
+            lhs(31,11)=clhs621;
+            lhs(31,12)=clhs344*clhs712 + clhs652;
+            lhs(31,13)=clhs344*clhs713 + clhs676;
+            lhs(31,14)=clhs344*clhs714 + clhs699;
+            lhs(31,15)=clhs715;
+            lhs(31,16)=clhs344*clhs786 + clhs740;
+            lhs(31,17)=clhs344*clhs787 + clhs759;
+            lhs(31,18)=clhs344*clhs788 + clhs777;
+            lhs(31,19)=clhs789;
+            lhs(31,20)=clhs344*clhs840 + clhs808;
+            lhs(31,21)=clhs344*clhs841 + clhs822;
+            lhs(31,22)=clhs344*clhs842 + clhs835;
+            lhs(31,23)=clhs843;
+            lhs(31,24)=clhs344*clhs873 + clhs855;
+            lhs(31,25)=clhs344*clhs874 + clhs864;
+            lhs(31,26)=clhs344*clhs875 + clhs872;
+            lhs(31,27)=clhs876;
+            lhs(31,28)=clhs345*clhs882;
+            lhs(31,29)=clhs345*clhs885;
+            lhs(31,30)=clhs345*clhs887;
+            lhs(31,31)=clhs343*(clhs877 + clhs883 + clhs886);
+
+
+    // Add intermediate results to local system
+    noalias(rLHS) += lhs * rData.Weight;
+}
+
+template <>
+void SymbolicStokes<SymbolicStokesData<2,3>>::ComputeGa
```

Comment:
```suggestion
void SymbolicStokes<TElementData>::GetValueOnIntegrationPoints(
    const Variable<double> &rVariable,
    std::vector<double> &rValues,
    const ProcessInfo &rCurrentProcessInfo)
```
Please follow the style guide.

Reply:
done.

Reply:
Done.